src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
-}
+  const arrOfKeys = Object.values(robot).reverse();
+  const arrOfValue = Object.keys(robot).reverse();
```

Comment:
You can use Object.entries instead of using two separate functions and two separate variables

Reply:
Yep, better to use entries to keep key and value tied together.